### PR TITLE
added all test for utils_plot + refactoring some similar functions (k…

### PR DESCRIPTION
**Pull request by davidmeunier79**
…ept for the sake of compatibility)

### CODE REVIEW

File: graphpype/tests/test_utils_plot.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+signals_matrix = np.random.rand(nb_nodes, nb_timepoints)
+
+try:
+    shutil.rmtree(tmp_dir)
+except OSError:
+    print("dir {} do not already exists".format(tmp_dir))
+
+try:
+    os.makedirs(tmp_dir)
+except OSError:
```
Comment (jasmainak) [inline graphpype/tests/test_utils_plot.py]: you should avoid `try-except` when you can ... it hides bugs.

Here for instance, you should use `os.path.exists`.

File: graphpype/tests/test_utils_plot.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    plot_cormat(plot_file, cor_mat=cor_mat)
+
+    assert os.path.exists(plot_file), "Error, {} was not created".format(
+        plot_file)
+
+
+def test_plot_ranged_cormat():
+    """
+    test plot_ranged_cormat
+    """
```
Comment (jasmainak) [inline graphpype/tests/test_utils_plot.py]: Docstring should be single line.

File: graphpype/tests/test_utils_plot.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    plot_hist(plot_file, data=cor_mat)
+
+    assert os.path.exists(plot_file), "Error, {} was not created".format(
+        plot_file)
+
+
+def test_plot_signals():
+    """
+    test plot_signals
+    """
```
Comment (jasmainak) [inline graphpype/tests/test_utils_plot.py]: same comment in all of them.

**Comment by davidmeunier79** [comment]
I have done modifications you suggested. I still issues with testing plot function in both test environments as they require the variable DISPLAY to be set. Any idea how to to that in the config files?

**Comment by davidmeunier79** [comment]
I have done modifications you suggested. I still issues with testing plot function in both test environments as they require the variable DISPLAY to be set. Any idea how to to that in the config files?

**Comment by jasmainak** [comment]
you need to use the 'agg' backend of `matplotlib`. See how we do it in `ephypype`. Also make the `matplotlib` imports nested.

**Comment by davidmeunier79** [comment]
yes, that seems to do the trick! Thanks

**Comment by jasmainak** [comment]
@davidmeunier79 good to go for me. I can't comment more. If anyone knows the code better, you can ask their feedback. Otherwise feel free to merge.

By the way, you should add a codecov badge in the README. See how we do it in `ephypype`

**Pull request merged**